Unable to run only sqlite tests

The contributing document says that tests can be run for specified databases by running `DB='sqlite3' npm test`.  Running this results in an error which is referencing the `oracledb` error:
```
$ DB='sqlite3' npm test

> knex@0.12.6 test /home/alex/projects/github/knex
> npm run lint && istanbul --config=test/.istanbul.yml cover node_modules/mocha/bin/_mocha -- --check-leaks -t 10000 -b -R spec test/index.js && npm run tape


> knex@0.12.6 lint /home/alex/projects/github/knex
> eslint src/**

Knex:warning - sqlite does not support inserting default values. Set the `useNullAsDefault` flag to hide this warning. (see docs http://knexjs.org/#Builder-insert).
Knex:warning - sqlite does not support inserting default values. Set the `useNullAsDefault` flag to hide this warning. (see docs http://knexjs.org/#Builder-insert).
Knex: run
$ npm install oracledb --save
Error: Cannot find module 'oracledb'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Client_Oracledb._driver (/home/alex/projects/github/knex/lib/dialects/oracledb/index.js:9:2912)
    at Client_Oracledb.initializeDriver (/home/alex/projects/github/knex/lib/client.js:9:13220)
    at Client_Oracledb.Client (/home/alex/projects/github/knex/lib/client.js:9:6366)
    at Client_Oracledb.Client_Oracle (/home/alex/projects/github/knex/lib/dialects/oracle/index.js:9:4239)
    at new Client_Oracledb (/home/alex/projects/github/knex/lib/dialects/oracledb/index.js:9:2124)
    at Knex (/home/alex/projects/github/knex/lib/index.js:9:3627)
    at Object.<anonymous> (/home/alex/projects/github/knex/test/unit/dialects/oracledb.js:6:20)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Object.Module._extensions.(anonymous function) [as .js] (/home/alex/projects/github/knex/node_modules/istanbul/lib/hook.js:109:37)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Suite.<anonymous> (/home/alex/projects/github/knex/test/index.js:36:3)
    at Object.create (/home/alex/projects/github/knex/node_modules/mocha/lib/interfaces/common.js:114:19)
    at context.describe.context.context (/home/alex/projects/github/knex/node_modules/mocha/lib/interfaces/bdd.js:44:27)
    at Object.<anonymous> (/home/alex/projects/github/knex/test/index.js:34:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Object.Module._extensions.(anonymous function) [as .js] (/home/alex/projects/github/knex/node_modules/istanbul/lib/hook.js:109:37)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at /home/alex/projects/github/knex/node_modules/mocha/lib/mocha.js:222:27
    at Array.forEach (native)
    at Mocha.loadFiles (/home/alex/projects/github/knex/node_modules/mocha/lib/mocha.js:219:14)
    at Mocha.run (/home/alex/projects/github/knex/node_modules/mocha/lib/mocha.js:487:10)
    at Object.<anonymous> (/home/alex/projects/github/knex/node_modules/mocha/bin/_mocha:459:18)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Object.Module._extensions.(anonymous function) [as .js] (/home/alex/projects/github/knex/node_modules/istanbul/lib/hook.js:109:37)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Function.Module.runMain (module.js:604:10)
    at runFn (/home/alex/projects/github/knex/node_modules/istanbul/lib/command/common/run-with-cover.js:122:16)
    at /home/alex/projects/github/knex/node_modules/istanbul/lib/command/common/run-with-cover.js:251:17
    at /home/alex/projects/github/knex/node_modules/istanbul/lib/util/file-matcher.js:68:16
    at /home/alex/projects/github/knex/node_modules/istanbul/node_modules/async/lib/async.js:52:16
    at /home/alex/projects/github/knex/node_modules/istanbul/node_modules/async/lib/async.js:361:13
    at /home/alex/projects/github/knex/node_modules/istanbul/node_modules/async/lib/async.js:52:16
    at done (/home/alex/projects/github/knex/node_modules/istanbul/node_modules/async/lib/async.js:246:17)
    at /home/alex/projects/github/knex/node_modules/istanbul/node_modules/async/lib/async.js:44:16
    at /home/alex/projects/github/knex/node_modules/istanbul/node_modules/async/lib/async.js:358:17
    at LOOP (fs.js:1758:14)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
=============================================================================
Writing coverage object [/home/alex/projects/github/knex/test/coverage/coverage.json]
Writing coverage reports at [/home/alex/projects/github/knex/test/coverage]
=============================================================================

=============================== Coverage summary ===============================
Statements   : 32.53% ( 1884/5792 )
Branches     : 13.48% ( 361/2678 )
Functions    : 10.54% ( 135/1281 )
Lines        : 33.23% ( 1760/5296 )
================================================================================
npm ERR! Test failed.  See above for more details.

```

--- a/test/index.js
+++ b/test/index.js
@@ -31,10 +31,13 @@ describe('Query Building Tests', function() {
   require('./unit/schema/oracledb')
 })
 
-describe('ExternalAuth ORACLE Tests', function() {
-  this.timeout(process.env.KNEX_TEST_TIMEOUT || 5000);
-  require('./unit/dialects/oracledb')
-})
+var config = require('./knexfile');
+if (config.oracle) {
+  describe('ExternalAuth ORACLE Tests', function() {
+    this.timeout(process.env.KNEX_TEST_TIMEOUT || 5000);
+    require('./unit/dialects/oracledb');
+  });
+}
 
 describe('Integration Tests', function() {
   this.timeout(process.env.KNEX_TEST_TIMEOUT || 5000);

